packages/test/src/index.js

```diff
@@ -0,0 +1,52 @@
+// Imports
+const express = require("express");
+const compression = require('compression')
+const cors = require('cors');
+const CONSTS = require('./utils/consts')
+
+// Import checkers
+const CommandsChecker = require('./tests/CommandsChecker')
+const ServicesChecker = require('./tests/ServicesChecker')
+const AppsChecker = require('./tests/AppsChecker')
+
+//Env Vars
+const ENV_VARS = process.env;
+
+// Setup server app
+const app = express();
+
+// Set the express middlewares
+app.use(express.static("public"));
+app.use(cors());
+app.use(compression());
+
+// Define the home route
+app.get("/", function (req, res) {
+  res.send("<h1>Canon Test!</h1>")
```

Comment:
Is your eslint plugin working in your IDE? I spotted this line missing an end `;`, but there may be a few other small things in these files.

Reply:
You are right. There was a problem with my eslint after some update. Solved.